### ISSUE TITLE
Voting: Allow switching between ID voting and non-ID voting once vote has started?  

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#622]
@kevincrepin @anettlinno I understand that it's not possible to change the voting settings generally after a vote has started (and see why not), but is there a reason why we don't allow switching from e-ID voting to non-ID voting mid-way, if the user wants to do so?  

In a test, it happened that the user set e-ID voting by accident (will explain how in the user tests follow-up later) and wanted to remove it so that I could vote (without e-ID) in his test. 

I don't know how often it would happen in real life that someone starts a vote with e-ID voting enforced and then wants to remove it, or vice versa, but I imagine it could happen sometimes (either because they made a mistake, or because they change their mind for some reason), and don't see that it would be a problem to allow this flexibility myself? It would just mean that some of the votes were recorded one way, and some another, but the votes would still all be cast on the same question? So no big deal democracy-wise, if the admin has knowingly chosen to make this change? (There could be a notification that double checks they understand the consequences?). 

If you agree that it would be good to allow this to be changed after voting started, then this is where the user was looking for it:

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/d8470118-3431-4c75-900a-0e87d1e5c952)

**Comment by anettlinno** [comment]
I think this needs more discussion and proper use cases.

What you explained might really happen -- that admin choses eID voting, but later it will be clear that some people cannot vote because they don´t have eID and/or they don´t need eID votes. It also might be that people are little bit nervous during the user tests and they didn´t have enough time to read and digest the options...

BUT the root-cause in my opinion is the fact that for the first-time user our voting system with voting rights option is not clear enough. I actually think that  we should explain that if they don´t choose eID or vote delegation, then by default there is a social voting... This does not come out anywhere for now... 

![image](https://github.com/citizenos/citizenos-fe/assets/45175679/7b5b628f-685c-40c2-a4cc-bd68f7221abc)

So, before giving admins an option to change voting rights, I would try to make it more clear in the process.

@ilmartyrk @BeccaMelhuish @kevincrepin WDYT?

**Comment by BeccaMelhuish** [comment]
@anettlinno yes this is exactly what happened actually! I was going to make a separate issue for that problem (I still might as I think there were other similar situations, need to check) but what happened is that the user thought he had to chose _either_ e-ID _or_ Delegation, and that he couldn't continue without one or the other. 

He was quite clear when selecting it that he didn't want eID voting, saying that we should have the option to not eID vote for things that aren't so formal (as he didn't realise that not selecting it would allow that). But as he was just trying out the platform for what it could do at that point, and he thought he didn't have a choice (without selecting delegation), he continued with it selected. (And he only found out that it was possible to set it up without eID because I later told him, otherwise he would have thought it wasn't possible, and would have decided it wasn't suitable for what he wanted to do). 

So yes I think there are a few things here: 
- Users assuming they have to select one thing per section, and when there are several options, are confused if it is a choice between the two or they can select either zero or several at once. 
- Users not understanding what it means if they don't select eID (i.e. it means voting without eID is possible)
- Users may have a need to switch from eID voting to non-ID voting - for example if they realise part-way through that some people can't use eID, as Anett said. (I think this may not be common, but I also think there's no harm in giving the option to edit it for when it does happen?).  

And I think whatever solution we do come up with for making the 'non-ID' voting, let's not call it 'social voting' as I think it's confusing/misleading? We know what we mean internally, but I don't think it makes sense actually (for the user, a participant's vote doesn't have much to do with their social media account, and they might have logged in with email anyway, not social media). We can find something clearer hopefully - will have a think.

**Comment by BeccaMelhuish** [comment]
Not super important to fix before the next tests, except the confusion that led him to keep eID selected despite not wanting it, but that's covered [here](https://github.com/citizenos/citizenos-fe/issues/665) now (with 'pre UX tests' label)

**Comment by kevincrepin** [comment]
I made a suggestion on this [issue](https://github.com/citizenos/citizenos-fe/issues/665) that **hopefully** makes this setup clearer.

**Comment by anettlinno** [comment]
Closing this issue as this will be solved with #665

**Comment by BeccaMelhuish** [comment]
@anettlinno Actually that wasn't the original issue, it was about whether we should allow it to change mid-way :)
Which I realise probably isn't a consideration for right now (just because of time restraints), but should we keep it open for consideration when we have time to?

**Comment by BeccaMelhuish** [comment]
Will re-open so it doesn't get missed (seems potentially important to me) but you can close again if you think not :)
